src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_FACTOR = 2;
+    private Node<K, V> []table;
+    private int size;
+    private int threshold;
+    private int capacity;
```

Comment:
please remove these fields, they only make the code extra complicated. We can always get capacity from another field - table, and threshold is used only once, in resize